src/models/bullet.js

```diff
@@ -67,9 +88,10 @@ nv.models.bullet = function() {
             var gEnter = wrapEnter.append('g');
             var g = wrap.select('g');
 
-            gEnter.append('rect').attr('class', 'nv-range nv-rangeMax');
-            gEnter.append('rect').attr('class', 'nv-range nv-rangeAvg');
-            gEnter.append('rect').attr('class', 'nv-range nv-rangeMin');
+            for(var i=0,il=rangez.length; i<il; i++){
+                gEnter.append('rect').attr('class', 'nv-range nv-range'+i+' nv-range'+legacyRangeClassNames[i]);
+            }
```

Comment:
If you have more than 3 ranges, you'll throw an index error here on legacyRangeClassNames...